src/helpers/generateFiles.ts

```diff
@@ -48,13 +49,16 @@ export function generateFiles(opts: {
     return [typesFileWithEnums];
   }
 
+  const allowImportingTsExtensions =
```

Comment:
In `prisma-json-types-generator`, i found so many edge cases in where this `allowImportingTsExtensions` logic failed that I decided to simply add a `importFileExtension` option and emit whatever the user set to it. I believe we should do the same here.

https://github.com/arthurfiorette/prisma-json-types-generator/blob/8c14de2abb6f4dc92126b474d46fdfc180a6102b/src/util/declaration-writer.ts#L22C22-L22C41

Reply:
Sounds good - I considered this but wanted to try the "magic" approach of doing a best effort guess. But makes sense to have it defined. 

Given we're doing a major version bump here, I think it makes sense to default to `.ts` here, any thoughts? Almost all new apps nowadays are using moduleResolution Bundler or NodeNext

Reply:
Most of our dependents are apps that have been running in prod for a while now. This new default would break for them all until they figure out that a different new default is now their problem. 

The default should be whatever that makes the generated output equal as it is today.